QualificationsRegister/Models/Exceptions/ForbiddenRequestException.cs

```diff
@@ -0,0 +1,7 @@
+namespace Ofqual.Common.RegisterAPI.Models.Exceptions
+{
+    public class ForbiddenRequestException : Exception
```

Comment:
We no longer need this. Confirmed with Toby that we'll use the Bad Request exception to deal with bad param inputs.